config/jetpack-cloud-development.json

```diff
@@ -10,6 +10,7 @@
 	"rtl": false,
 	"login_url": "/connect",
 	"logout_url": "https://jetpack.com",
+	"signup_url": "https://wordpress.com/jetpack/connect",
```

Comment:
On WordPress.com, this config variable points to `/start` which is the route that allows users to create a new WordPress.com site.